.gitmodules

```diff
@@ -7,27 +7,12 @@
 [submodule "lib/federation-service/federation_service"]
   path = lib/federation-service/federation_service
   url = https://github.com/CanDIG/federation_service
-[submodule "lib/cnv-service/candig_cnv_service"]
-  path = lib/cnv-service/candig_cnv_service
-  url = https://github.com/CanDIG/candig_cnv_service
-[submodule "lib/rnaget/rnaget_service"]
-  path = lib/rnaget/rnaget_service
-  url = https://github.com/CanDIG/rnaget_service
-[submodule "lib/datasets/datasets_service"]
-  path = lib/datasets/datasets_service
-  url = https://github.com/CanDIG/datasets_service
 [submodule "lib/chord-metadata/chord_metadata_service"]
   path = lib/chord-metadata/chord_metadata_service
   url = https://github.com/CanDIG/katsu.git 
-[submodule "lib/cancogen-dashboard/cancogen_dashboard"]
-  path = lib/cancogen-dashboard/cancogen_dashboard
-  url = https://github.com/CanDIG/cancogen_dashboard.git
 [submodule "lib/candig-data-portal/candig-data-portal"]
   path = lib/candig-data-portal/candig-data-portal
   url = https://github.com/CanDIG/candig-data-portal.git
 [submodule "lib/opa/opa"]
   path = lib/opa/opa
   url = https://github.com/CanDIG/candig-opa.git
-[submodule "lib/graphql/GraphQL-interface"]
```

Comment:
I think we might be bringing this one back soon...?

Reply:
The graphql module was never in the list of modules in example.env and also rnaget is not a fully usable module (it might also make sense to bundle it with htsget). Both repos will require refactoring and can be reintroduced when needed in the future.